addons/godot-next/resources/behavior.gd

```diff
@@ -0,0 +1,60 @@
+# author: xdgamestudios
+# license: MIT
+# description: This is an abstract base "Behavior" class for use in the "Behaviors" node class.
+#              "Behaviors" manages "Behavior" resources and calls notification methods that the Behavior implements.
+# usage: 
+# - Supported notifications:
+#     _ready() -> void
+#     _process(p_delta: float) -> void
+#     _physics_process(delta: float) -> void:
+#     _input(event: InputEvent) -> void:
+#     _unhandled_input(event: InputEvent) -> void:
+#     _unhandled_key_input(event: InputEventKey) -> void:
+#     Note:
+#     - Notifications, if present, are automatically triggered by the owner class.
+#     - If the behavior is disabled its notifications will not be processed. 
+# - Enabled Property:
```

Comment:
Actually, rather than keeping these property comments up here, let's just put them on a line of their own above their declarations.